xml/System.Threading/CountdownEvent.xml

```diff
@@ -745,7 +745,10 @@
           <paramref name="signalCount" /> is less than or equal to 0.</exception>
         <exception cref="T:System.InvalidOperationException">
           <see cref="P:System.Threading.CountdownEvent.CurrentCount" /> + <paramref name="signalCount" /> is equal to or greater than <see cref="F:System.Int32.MaxValue" />.
-        </exception>
+
+-or-
+
+The current instance is already set.</exception>
```

Comment:
This comment applies to `AddCount`, which throws when the event is signaled. `TryAddCount` just returns false, so this doesn't apply here.

Reply:
Removed.